mws/sitesmanagement/cronjobs.py

```diff
@@ -45,63 +47,67 @@ def on_failure(self, exc, task_id, args, kwargs, einfo):
                      "The parameters passed to the task were: %s\n\nThe traceback is:\n%s\n", task_id, args, einfo)
 
 
+REMINDER_RENEWAL_SUBJECT = "The annual charge for your managed web server is due on '%s'"
+REMINDER_RENEWAL_BODY = (
+    "You are receiving this message because your email address, or an email alias that includes you as a recipient, "
+    "has been configured as the contact address for the UIS Managed Web Server '%s'.\n\n"
+    "The annual charge for your managed web server '%s' is due on '%s'. "
+    "Unless you tell us otherwise we will automatically issue an invoice for this on '%s' "
+    "based on information from the most recent purchase order you have given us. "
+    "Please use the web control panel (under 'billing settings') to check that this information is still current. "
+    "If you want to amend your purchase order you can upload a new one. "
+    "Your site may be cancelled if we can't successfully invoice for it.\n\n"
+    "If you no longer want you site then please either cancel it now (under 'edit the MWS profile'), "
+    "or mark it 'Not for renewal' in which case it will be automatically cancelled on '%s'."
+)
+
+
 @shared_task(base=FinanceTaskWithFailure)
 def send_reminder_renewal():
-    today = timezone.now().date()
-    # Billings of sites that haven't been canceled (end_date is null), that hasn't expressed to want to cancel
-    # their subscription, and that started in the previous month of the current one of a previous year
-    renewal_sites_billing = Billing.objects.filter(site__start_date__month=today.month+1 if today.month != 12 else 1,
-                                                   site__start_date__lt=date(today.year, 1, 1),
-                                                   site__end_date__isnull=True,
-                                                   site__subscription=True)
-    for billing in renewal_sites_billing:
+    today = now().date()
+
+    def renewal_sites_billing(lower, upper=0):
```

Comment:
Since the only variable captured by this closure is "today", can we not hoist it out to be a global function?